### PR TITLE
Check for mutually exclusive flags

### PR DESCRIPTION
**Pull request by umohnani8**
Error out if more than one mutually exclusive flags are passed
in to kpod ps

Addresses issue @rhatdan raised in https://github.com/projectatomic/libpod/pull/123#pullrequestreview-82995550

Signed-off-by: umohnani8 <umohnani@redhat.com>

### CODE REVIEW

**Comment by umohnani8** [comment]
@rhatdan @mheon PTAL

**Comment by mheon** [comment]
LGTM

File: cmd/kpod/ps.go [hunk 1]
```diff
@@ -213,6 +212,32 @@ func psCmd(c *cli.Context) error {
...
 
+// checkFlagsPassed checks if mutually exclusive flags are passed together
+func checkFlagsPassed(c *cli.Context) error {
+	// latest, and last are mutually exclusive.
+	if c.Int("last") >= 0 && c.Bool("latest") {
+		return errors.Errorf("last and latest are mutually exclusive")
+	}
+	// quiet, size, namespace, and format with Go template are mutually exclusive
+	flags := 0
+	if c.Bool("quiet") {
```
Comment (TomSweeneyRedHat) [inline cmd/kpod/ps.go]: Do we have tests for this already?

**Comment by TomSweeneyRedHat** [comment]
FWIW @umohnani8 looks like more fun with "unable to find image 'registry.fedoraproject.org/fedora:26' locally", not related to these changes.

File: cmd/kpod/ps.go (hunk 1 above)
Comment (umohnani8) [reply cmd/kpod/ps.go]: Added tests.

**Comment by umohnani8** [comment]
@rhatdan PTAL

**Comment by rhatdan** [comment]
@rh-atomic-bot r+

**Pull request closed without merging**